utilities/utils.py

```
@@ -106,6 +108,76 @@ def _fetch_and_store_cacert(
+
+    for csv in ClusterServiceVersion.get(client=ocp_admin_client, namespace=namespace):
+        if csv.instance and csv.instance.spec:
+            display_name = csv.instance.spec.displayName
+            version_str = csv.instance.spec.version
+
+            if display_name and "Migration Toolkit for Virtualization" in display_name:
+                mtv_version = version_str
+                break
+
+    if not mtv_version:
+        raise RuntimeError("MTV operator version not found")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)